### PR TITLE
DUOS-1296[risk=no]Update api url in configs

### PR DESCRIPTION
**Pull request by raejohanek**
SCOPE:
remove /api from config and add to urls in ajax
merge must be accompanied by changes in terra helm and terra helm files as well as changes to google configs
ADDRESSES:
https://broadworkbench.atlassian.net/browse/DUOS-1296

Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] PR is labeled with a Jira ticket number and includes a link to the ticket
- [ ] PR is labeled with a security risk modifier [no, low, medium, high] 
- [ ] PR describes scope of changes

In all cases:

- [ ] Get a minimum of one thumbs worth of review, preferably 2 if enough team members are available
- [ ] Get PO sign-off for all non-trivial UI or workflow changes
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by rushtong** [comment]
Will require an update to the configmaps in broad-duos-dev and broad-duos-prod when this is eventually merged. Please coordinate with me so we can ensure no disruption in the deployments.

**Comment by rushtong** [comment]
Look here as well for simplification: https://github.com/DataBiosphere/duos-ui/blob/develop/src/pages/Status.js#L30

**Comment by rushtong** [comment]
~This needs an update: https://github.com/DataBiosphere/duos-ui/blob/develop/src/libs/ajax.js#L189~ ... sorry, not a valid comment

**Comment by rushtong** [comment]
This line needs an update: https://github.com/DataBiosphere/duos-ui/blob/develop/src/pages/DulResultRecords.js#L67

**Review by rushtong — CHANGES_REQUESTED** [review]
Looking good, please see comments on the PR.

File: src/pages/DulResultRecords.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ class DulResultRecords extends Component {
       election: electionReview.election,
       dataUse,
       dul: electionReview.election.dataUseLetter,
-      downloadUrl: await Config.getApiUrl() + 'consent/' + electionReview.consent.consentId + '/dul',
+      downloadUrl: await Config.getApiUrl() + '/api/consent/' + electionReview.consent.consentId + '/dul',
```
Comment (rushtong) [inline src/pages/DulResultRecords.js:67]: This is legacy code that will be going away at some point, but it illustrates why I like to have all url construction localized in ajax.js. When we distribute this kind of functionality across the pages, refactoring is harder.

Comment (raejohanek) [reply src/pages/DulResultRecords.js:67]: yeah I don't believe this download url is actually used anywhere

**Review by rushtong — APPROVED** [review]
Looks good, thank you for the changes. Let's coordinate a time to merge the related PRs.

**Pull request merged**